src/components/Tile/Tile.scss

```diff
@@ -10,6 +10,41 @@
   color: $tile-text-color;
   font-size: $tile-font-size;
 
+  .control {
+    background-color: transparent;
+    background-size: 24px;
+    border: none;
+    height: 24px;
+    width: 24px;
+
+    position: absolute;
+    top: 4px;
+    opacity: 0;
+    z-index: 1;
+    transition: opacity 100ms ease-out;
+    cursor: pointer;
+  }
+
+  .tile-img-container:hover > .control {
+    opacity: 1;
+  }
+
+  .control-block:-moz-dir(ltr) {
+    right: 4px;
+  }
+
+  .control-block {
+    background-image: -moz-image-rect(url(../img/controls.svg), 0, 192, 32, 160);
+  }
+
+  .control-block:hover {
+    background-image: -moz-image-rect(url(../img/controls.svg), 0, 224, 32, 192);
+  }
+
+  .control-block:hover:active {
+    background-image: -moz-image-rect(url(../img/controls.svg), 0, 256, 32, 224);
```

Comment:
When I did a bit of searching, I couldn't find a standardized equivalent. The MDN docs also say it isn't standard: https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-image-rect

@k88hudson mentioned in the future we can use font icons which will be a better alternative to this (https://fortawesome.github.io/Font-Awesome/ I think?)